libs/wingsdk/src/std/array.ts

```diff
@@ -0,0 +1,65 @@
+// These classes are used by Wing to provide JSII subsets of the JS Array class.
+// They should not be consumed directly by users.
+// TODO: These should be interfaces, currently Wing does not support interface JSII imports
+
+/**
+ * Mutable Array
+ */
+export class MutArray {
```

Comment:
`MutArray` should extend `Array`...

(we can add this in a subsequent PR)

Reply:
This is hard because JSII gets mad if you try to extend Array (it assumes the global array).

To support this, I decided to rename these to ImmutableArray and MutableArray, there's technically no reason the class name needed to match.